api/profile/profileRouter.js

```diff
@@ -415,7 +417,7 @@ router.post('/availability/:profile_id', (req, res, next) => {
   const { profile_id } = req.params;
   const { accepting_new_mentees } = req.body;
   axios
-    .post(`${process.env.DS_API_URL}/update/mentor/${profile_id}`, {
+    .post(`${baseURL}/update/mentor/${profile_id}`, {
```

Comment:
It's always satisfying when code is nice and readable. Great work. Changing `process.env.DS_API_URL` to `baseURL` will help future developers to understand and familiarize themselves with the codebase.